modules/EnsEMBL/Draw/GlyphSet/genetree.pm

```diff
@@ -746,7 +746,7 @@ sub features {
     $f->{'_height'}             = 12 * log($f->{'_collapsed_count'});
     $f->{'_genome_dbs'}         = \%genome_dbs;
     $f->{'_genes'}              = \%genes;
-    $f->{'_leaves'}             = \%leaves;
```

Comment:
For what it's worth, I've checked the webcode repos and failed to find `_leaves` being used as a hash key anywhere other than the `EnsEMBL::Draw::GlyphSet::genetree` module.